lib/src/RedisRemoteSaiInterface.cpp

```diff
@@ -827,6 +827,106 @@ sai_status_t RedisRemoteSaiInterface::waitForObjectTypeGetAvailabilityResponse(
     return status;
 }
 
+sai_status_t RedisRemoteSaiInterface::queryAattributeCapability(
+        _In_ sai_object_id_t switchId,
+        _In_ sai_object_type_t objectType,
+        _In_ sai_attr_id_t attrId,
+        _Out_ sai_attr_capability_t *capability)
+{
+    SWSS_LOG_ENTER();
+
+    auto switch_id_str = sai_serialize_object_id(switchId);
+    auto object_type_str = sai_serialize_object_type(objectType);
+
+    auto meta = sai_metadata_get_attr_metadata(objectType, attrId);
+
+    if (meta == NULL)
+    {
+        SWSS_LOG_ERROR("Failed to find attribute metadata: object type %s, attr id %d", object_type_str.c_str(), attrId);
+        return SAI_STATUS_INVALID_PARAMETER;
+    }
+
+    const std::string attr_id_str = meta->attridname;
+
+    const std::vector<swss::FieldValueTuple> entry =
+    {
+        swss::FieldValueTuple("OBJECT_TYPE", object_type_str),
+        swss::FieldValueTuple("ATTR_ID", attr_id_str)
+    };
+
+    SWSS_LOG_DEBUG(
+            "Query arguments: switch %s, object type: %s, attribute: %s",
+            switch_id_str.c_str(),
+            object_type_str.c_str(),
```

Comment:
in all other places we tray to keep format prefixed by str like "strObjectType", coult you keep that also in all your variables?

Reply:
Renamed to follow the suggested naming convention for the variables introduced.